openest/test/curves/test_bounding.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import numpy as np
+
+from openest.curves.bounding import *
+
+## 1-D test
+def test_1d():
+    polytope = [(.25,), (.75,)]
+    iter = facets(polytope)
```

Comment:
Minor thing but using `iter` as a variable name means that we're clobbering the base `iter()` function. It feels and reads kinda weird. I don't think this has a side effect right now so this isn't essential.

If you do rename this, be sure to grab the other `iter`s below.